app/src/main/java/com/kodeco/android/coordplot/MainActivity.kt

```diff
@@ -12,7 +14,14 @@ class MainActivity : ComponentActivity() {
 
     override fun onCreate(savedInstanceState: Bundle?) {
         installSplashScreen()
-        val repository = CountryRepositoryImpl(apiService)
+
+        val db = Room.databaseBuilder(
```

Comment:
You can move this code for building the database to CountryInfoDatabase file to follow the Single Responsibility Principle. Activities should mostly contain any UI related code.